docs/src/Status.doc.js

```diff
@@ -0,0 +1,86 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { Status } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="Status"
+    description="Status is an indicator with an icon that provides information to a user."
+    defaultCode={`
+<Flex>
+  <Status type='ok' title="OK" subtext="Updated 2 days ago" />
+</Flex>
+`}
+    pilot
+  />,
+);
+
+// Specifying Component gives auto warnings about missing props
+card(
+  <PropTable
+    Component={Status}
+    props={[
+      {
+        name: 'type',
+        type: `'unstarted' | 'inProgress' | 'halted' | 'ok' | 'problem' | 'canceled' | 'warning'`,
+        required: true,
+        description: 'The type of status to display',
+      },
+      {
+        name: 'title',
+        type: 'string',
+        required: true,
+        description: 'A label to reinforce the meaning of the status icon',
```

Comment:
this needs a link to the localization section, like "See localization to learn more"